packages/core/src/cart/headless-cart/headless-cart.ts

```diff
@@ -0,0 +1,141 @@
+interface BaseFieldFragment {
+    value: number;
+}
+
+export interface HeadlessLineItem {
+    name: string;
+    entityId: string;
+    quantity: number;
+    productEntityId: number;
+    brand: string;
+    couponAmount: {
+        value: number;
+    };
+    discountedAmount: {
+        value: number;
+    };
+    discounts: Array<{
+        discountedAmount: {
+            value: number;
+        };
+        entityId: string;
+    }>;
+    extendedListPrice: {
+        value: number;
+    };
+    extendedSalePrice: {
+        value: number;
+    };
+    imageUrl: string;
+    isTaxable: boolean;
+    listPrice: {
+        value: number;
+    };
+    originalPrice: {
+        value: number;
+    };
+    salePrice: {
+        value: number;
+    };
+    sku: string;
+    url: string;
+    variantEntityId: number;
+    selectedOptions: Array<{
+        __typename: string;
```

Comment:
Do we need this?

Reply:
This field is available by default. So I added it to interface

Reply:
If we don't need it, maybe we shall omit it.